### PR TITLE
format change only on quick start install section

### PR DESCRIPTION
**Pull request by linsun**

### CODE REVIEW

**Comment by geeknoid** [comment]
Deploy preview for *preliminary-istio* failed.

Built with commit 3953d3ed129078837840a1fa740b5ab09ddc2ac5

https://app.netlify.com/sites/preliminary-istio/deploys/5b0ee74a3672df15d6ec74d5

**Comment by geeknoid** [comment]
Deploy preview for *preliminary-istio* failed.

Built with commit 71a1c2e78ce8f6da330ee7c68949001646334071

https://app.netlify.com/sites/preliminary-istio/deploys/5b0ee77a1f12b7068c416603

File: content/docs/setup/kubernetes/quick-start.md [hunk 1]
```diff
@@ -254,31 +254,25 @@ For example, run the following command on a MacOS or Linux system:
...
 
     ```command
     $ kubectl apply -f install/kubernetes/istio-demo.yaml
     ```
+_**OR**_
 
-    _**OR**_
-
-    b) Quick install Istio and enforce mutual TLS authentication between sidecars by default. Use this option only on a fresh kubernetes cluster where newly deployed workloads are guaranteed to have Istio sidecars installed.
+b)  Quick install Istio and enforce mutual TLS authentication between sidecars by default. Use this option only on a fresh kubernetes cluster where newly deployed workloads are guaranteed to have Istio sidecars installed.
```
Comment (geeknoid) [inline content/docs/setup/kubernetes/quick-start.md]: Quickly install

File: content/docs/setup/kubernetes/quick-start.md [hunk 2]
```diff
@@ -254,31 +254,25 @@ For example, run the following command on a MacOS or Linux system:
...
 
-    a)  Quick install Istio using without enabling [mutual TLS authentication](/docs/concepts/security/mutual-tls/) between sidecars.
-    Choose this option for clusters with existing applications, applications where services with an
-    Istio sidecar need to be able to communicate with other non-Istio Kubernetes services, and
-    applications that use [liveness and readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/),
-    headless services, or StatefulSets.
+a)  Quick install Istio using without enabling [mutual TLS authentication](/docs/concepts/security/mutual-tls/) between sidecars. Choose this option for clusters with existing applications, applications where services with an Istio sidecar need to be able to communicate with other non-Istio Kubernetes services, and applications that use [liveness and readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/), headless services, or StatefulSets.
```
Comment (geeknoid) [inline content/docs/setup/kubernetes/quick-start.md]: Quickly install Istio without enabling

File: content/docs/setup/kubernetes/quick-start.md [hunk 3]
```diff
@@ -254,36 +254,32 @@ For example, run the following command on a MacOS or Linux system:
...
 
-    a)  Quick install Istio using without enabling [mutual TLS authentication](/docs/concepts/security/mutual-tls/) between sidecars.
-    Choose this option for clusters with existing applications, applications where services with an
-    Istio sidecar need to be able to communicate with other non-Istio Kubernetes services, and
-    applications that use [liveness and readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/),
-    headless services, or StatefulSets.
+a)  Quickly install Istio without enabling [mutual TLS authentication](/docs/concepts/security/mutual-tls/) between     sidecars. Choose this option for clusters with existing applications, applications where services with an Istio sidecar need to be able to communicate with other non-Istio Kubernetes services, and applications that use [liveness and readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/), headless services, or StatefulSets.
```
[the remark this answers is not in the document]
Comment (linsun) [reply content/docs/setup/kubernetes/quick-start.md]: Thanks, I had slightly trouble with Quickly install too.

File: content/docs/setup/kubernetes/quick-start.md [hunk 4]
```diff
@@ -254,36 +254,32 @@ For example, run the following command on a MacOS or Linux system:
...
+   __*OR*__
 
-    d)  [Use Helm and Tiller to manage the Istio deployment](/docs/setup/kubernetes/helm-install/#option-2-install-with-helm-and-tiller-via-helm-install).
+d)  [Use Helm and Tiller to manage the Istio deployment](/docs/setup/kubernetes/helm-install/#option-2-install-with-helm-and-tiller-via-helm-install).
 
 ## Verifying the installation
 
-1.  Ensure the following Kubernetes services are deployed: `istio-pilot`, `istio-ingress`,
-`istio-policy`, `istio-telemetry`, `prometheus`.
+1.  Ensure the following Kubernetes services are deployed: `istio-pilot`, `istio-ingressgateway`, `istio-egressgateway`,
```
Comment (gyliu513) [inline content/docs/setup/kubernetes/quick-start.md]: two `istio-policy` here

Comment (linsun) [reply content/docs/setup/kubernetes/quick-start.md]: good catch and i found istio-policy isn't even exposed as a service

**Comment by linsun** [comment]
can't get the deploy/netlify test to pass due to rule 36:  https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md036---emphasis-used-instead-of-a-heading

I suggest we loose the rule 36 checking, open to other suggestions,  @geeknoid

**Pull request merged**